### PR TITLE
Setting-And-UI Update-->Pre Code Driven UI

### PR DESCRIPTION
**Pull request by dustinwloring1988**
This will create a new scene for the code driven UI to be started in.
*Started to change some file to be more code driven friendly.
untitled2

### CODE REVIEW

File: Assets/Scripts/Settings/GameSettings.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿public class GameSettings 
+{
+	//this is for saving in the furture
+
+	public bool isVisableFPS;
+	public bool isVisableLangage;
+	public bool isFullscreen;
+	public float masterVolume;
+	public float musicVolume;
+	public int quilityIndex;
```
Comment (bjubes) [inline Assets/Scripts/Settings/GameSettings.cs]: Quality?

**Comment by dustinwloring1988** [comment]
Fixed

**Comment by Dormanil** [comment]
Okay, first off: Why are there changes in the original scene, when you have made a new Scene for the code-driven UI to be in? And why was the trading scene completely removed by you? I doubt that was a part of this feature you were implementing.

Then next up:

```c#
public class GameSettings 
{
	//this is for saving in the furture

	public bool isVisableFPS;
	public bool isVisableLangage;
	public bool isFullscreen;
	public float masterVolume;
	public float musicVolume;
	public int quilityIndex;
	public int resultionIndex;
	public int vSyncIndex;
	public int antiAliasingindex;
}
```

Please take a look at [CONTRIBUTING.md](https://github.com/TeamPorcupine/ProjectPorcupine/blob/master/CONTRIBUTING.md). We have already established that indentation is to be made with 4 spaces. Additionally, comment markers `//` and the comment text are to be separated by a space. Comments are to end with a period. Finally, `antiAliasingindex` is not properly camelCased, it should be `antiAliasingIndex`.

Then, please, _please_ take a look at your spelling. It's `future, visible, quality, resolution`, and that is only the errors in that particular code snippet.

PS: Yes, I truly am pedantic, but _someone_ has to be that way, otherwise other people are "fixing" spelling and code style errors that snuck into the master branch until the end of this project.

**Comment by dustinwloring1988** [comment]
OK fixed the spelling. and i kept his trading commit it's in the code driven scene. The next submit i'm working on will have more code driven UI. **I don't wanna be changing multiply scenes it will break them.**

**Pull request closed without merging**